### PR TITLE
Allow enabling aarch64 and ppc64le using default provider

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [x] Added a ``news`` entry

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
So does this automatically add azure if we have an aarch64 selector somewhere?

**Comment by isuruf** [comment]
No, but this adds azure if
```
provider:
   linux_ppc64le: default
```
or

```
provider:
   linux_ppc64le: azure
```
is there.

**Comment by mariusvniekerk** [comment]
ah okay, it makes azure the default, makes sense

File: conda_smithy/configure_feedstock.py [hunk 1]
```diff
@@ -340,9 +340,6 @@ def _collapse_subpackage_variants(list_of_metas):
     _trim_unused_zip_keys(used_key_values)
     _trim_unused_pin_run_as_build(used_key_values)
 
-    print("top_level_loop_vars", top_level_loop_vars)
```
Comment (mariusvniekerk) [inline conda_smithy/configure_feedstock.py]: I've found this logging quite useful when running smithy locally

Comment (isuruf) [reply conda_smithy/configure_feedstock.py]: They should be optional and enabled using a flag. I've restored them for now

**Pull request merged**